components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -282,6 +282,25 @@
             "type": "boolean",
             "deprecationMessage": "The 'experimentalNetwork' property is deprecated.",
             "description": "Experimental network configuration in workspaces (deprecated). Enabled by default"
+        },
+        "coreDump": {
+            "type": "object",
+            "description": "Configure the default action of certain signals is to cause a process to terminate and produce a core dump file, a file containing an image of the process's memory at the time of termination. Disabled by default.",
+            "deprecationMessage": "The 'coreDump' property is experimental.",
+            "additionalProperties": false,
+            "properties": {
+                "enabled": {
+                    "type": "boolean"
+                },
+                "cur": {
```

Comment:
Why not simply call this `soft-limit` (instead of `cur`) and rename `max` to `hard-limit` as well?  That would match `ulimit` documentation, make it easy to understand for people who know about `ulimit`, and make it easy to search for information on it (given a simple documentation that says "These attributes map to `ulimit -c`").  (`cur` doesn't really mean much, I mean, it is short for `current` but what does _that_ mean in a _config_ file?

Reply:
@david-bakin sure, done